### ISSUE TITLE
Fix FP S6747 (`no-unknown-property`): Don't raise issues from `eslint-plugin-react/no-unknown-property` when react is not used

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4634]
When using other frameworks such as [qwik](https://qwik.dev/), [StencilJS](https://stenciljs.com/) or [preact](https://preactjs.com/) these rules should not apply

## qwik

```tsx
import { component$ } from "@builder.io/qwik";

export default component$(() => {
  return (
    <header class={`${styles.header_header} ${styles.header_sticky}`}>
  )
}
```

## Stencil

```tsx
import {Component, Event, EventEmitter, h, Host, Prop} from '@stencil/core'

export class Foo {
  render() {
    return (
      <div 
        class={`item-wrapper ${this.errorClass}`} {/*FP*/}
      ></div>
    )
  }
}
```

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-22
